### ISSUE TITLE
Logging on with some generative doggerel

### ISSUE THREAD

**Issue by rmrenner** [NaPoGenMo/NaPoGenMo2017#6]
My NaPoGenMo entry uses the [CMU Pronouncing Dictionary](http://www.speech.cs.cmu.edu/cgi-bin/cmudict) to pick 2 syllable words according to their stressed vowels & stress pattern and then it slots them into a template grown from an L-System.

I was thinking that by creating irregular but self-similar patterns of stress and vowel sounds, rather than rhyme, I might get something like Gertrude Stein's "If I Told Him":

> If I told him would he like it. Would he like it if I told him.
> Would he like it would Napoleon would Napoleon would would he like it.

In retrospect, choosing a few pre-existing phrases and then sub-stringing them would probably work better for replicating that style. Since I'm strictly using (mostly trochaic) two syllable words, what I have instead is something that makes chants and tongue-twisters:

> **Dittmer Clifton Upson Ticker Finished**
> 
> Hipsters burger skillman kurland yeargin,
> dearmon glistened murtha winney merkin,
> gibbon hurry herbel gerber lichtman,
> merchant's mizell scheurer lipschutz durban.
> 
> Furtive burkman dearly hermance printers,
> gergen syrup(1) burbles herwitz flurries,
> prison's wursthaus sneary curtsy whidby,
> gerbig hertzler(1) learner whistling(1) fernberg.
> 
> Verdes girsky tilney ernests frickey,
> kirby bertholf ruggles citric murrelet,
> cristal kirkey turbos turner philip(1),
> bermans hitting kerbel fissell kirkner.
> 
> Girdle virkler(1) ridden stermer pillows,
> erma wickey burness skirted sturgeons,
> fish's purdie gilchrist bergfeld cissna,
> burchell ferger version rigby perlow.

**Comment by enkiv2** [comment]
Is (1) a sense indicator?

**Comment by rmrenner** [comment]
Yup! I haven't bothered stripping them out yet, but a number in parentheses indicates an alternate pronunciation for that word.

**Comment by hugovk** [comment]
See https://raw.githubusercontent.com/aparrish/pronouncingpy/master/pronouncing/cmudict-0.7b (3.54 MB)

**Comment by hugovk** [comment]
Is the source code for this up somewhere?

**Comment by rmrenner** [comment]
Yup! Here's the repository: [NaPoGenMo2017](https://github.com/rmrenner/NaPoGenMo2017).

Since the first draft, I've switched over to using the "pronouncing" python library and I've adjusted the script so that it now picks words of varying length.

Sample output:

> **megabucks been dusting kunka chen's with**
> 
> arrowhead arquette o'hara melle than
> specker second megabucks it's wendt the
> them good pairs o'dell macbeth carvell if
> terre_haute credentials debonair was
> 
> teleflex chimento scare le slugged le
> nels forgetting clenched celeb direction
> ventritex been pet louella delp was
> recompense regressions jed to hefty
